sos/report/plugins/virsh.py

```diff
@@ -46,6 +45,9 @@ def setup(self):
         for subcmd in subcmds:
             self.add_cmd_output('%s %s' % (cmd, subcmd), foreground=True)
 
+        self.add_cmd_output("%s list --all" % cmd,
```

Comment:
Please add `foreground=True` parameter like before (I *think* the cmd can hang otherwise..?).

Reply:
Updated